docs/application/vstools-mac/overview.md

```diff
@@ -1,12 +1,14 @@
-# Installing Visual Studio for Mac Extension for Tizen
+# Visual Studio for Mac Extension for Tizen
+
+Visual Studio for Mac Extension for Tizen enables you to develop Tizen aplications easily using Visual Studio Code.
 
 ## Prerequisites
 
 To work with Visual Studio for Mac Extension for Tizen, your computer must have:
 
 - At least 5.6 GB of available disk space
 - macOS Mojave 10.14: Community, Professional, and Enterprise.	
-- Visual Studio Mac 2019 (Community, Professional, Enterprise) Version 8.2 and above.
+- Visual Studio Mac 2019 (Community, Professional, Enterprise) Version 8.2 and higher.
```

Comment:
Visual Studio 2019 for Mac is an official name.

Reply:
Done